storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MemcacheLongStore.scala

```diff
@@ -34,9 +35,16 @@ import com.twitter.bijection.netty.ChannelBufferBijection
  */
 object MemcacheLongStore {
   private implicit val cb2ary = ChannelBufferBijection
+  private implicit val b2ary = new Bijection[Buf, Array[Byte]] {
```

Comment:
We have `AbstractBijection` for this. A little clearer than extending the trait.